katz/src/main/kotlin/katz/data/Option.kt

```diff
@@ -31,6 +31,7 @@ sealed class Option<out A> : OptionKind<A> {
     companion object : OptionMonad, GlobalInstance<Monad<Option.F>>() {
         inline fun <reified A : Any> fromNullable(a: A?): Option<A> = if (a != null) Option.Some(a) else Option.None
         operator fun <A> invoke(a: A): Option<A> = Option.Some(a)
+        inline fun <reified T: Any> empty() = Option.fromNullable<T>(null)
```

Comment:
Isn't that just the same as returning `None`? Also this is part of the interface for `MonadFilter` which is one of the MTL typeclasses which we don't have yet.